src/components/K8s.vue

```diff
@@ -87,21 +89,22 @@ export default {
     },
     handleCheckbox(event, name) {
       const status = event.target.checked;
-      const fullSourceName = `./resources/darwin/bin/${name}`
+      const fullSourceName = `${startingDirectory}/resources/darwin/bin/${name}`
       const fullTargetName = `/usr/local/bin/${name}`
       if (!(name in this.symlinks)) {
         console.error(`No setting for symlink ${name}`)
       } else if (status) {
-        if (this.filePresentAndNotSymLink(fullSourceName)) {
+        if (this.safeStat(fullTargetName)) {
           console.log(`Already have /usr/local/bin/${name}`);
         } else {
-          this.doCommand(`ln -s ${fullSourceName} ${fullTargetName}`);
+          if (!this.safeStat(fullSourceName)) {
+            console.error(`Can't find file ${fullSourceName}`);
+          } else {
+            this.doCommand(`ln -s ${fullSourceName} ${fullTargetName}`);
```

Comment:
This might be an issue if the cwd includes a space, such as in `Rancher Desktop.app` on mac.  It might be better to use [node APIs](https://nodejs.org/api/fs.html#fs_fspromises_symlink_target_path_type) for it instead.

Reply:
Thanks, didn't know that existed.